### DISCUSSION TITLE
Close chat

### DISCUSSION THREAD

**Discussion by salsero72** [LiveHelperChat/livehelperchat#1851] [General]
Hello
Whenever a chat is done and the user does not need replies anymore, the operator has 2 choices.
1. Do nothing
2. Close the chat

Then
1. If you do not do anything with the chat, as time passes and chats gets opened and left done, this happens: A new user opening a chat will see something like "You are the 82th in line." Even if no chat is pending. And this may discourage the user that will probably leave.
2. Instead If you close the chat the user that opened it cannot reopen the chat any more. Even if it closes the browsers and reopen. So returning users can no longer open chat and ask for help.

To circumvent number 2, you can go to Chat Configuration > Misc and set "Enable users to reopen closed chat" to true but then you cannot block annoying users.

Am i missing something?

**Comment by remdex** [comment]
Sorry, but I don't understand your scenario at all...

* How can chat be in 82th if this number is checked only for pending chats.
* How can't he ask for help? he just have to start another chat.
